src/it/MCOMPILER-366/verify.groovy

```diff
@@ -19,6 +19,6 @@
 
 buildLog = new File( basedir, 'build.log' ).text;
 
-assert buildLog.contains("[WARNING] Filename-based automodules detected on the module-path:")
+assert buildLog.contains("[WARNING] Filename-based automodules detected on the module path:")
```

Comment:
Why not keep `module-path` like `Filename-based` is?
It's the option used by both - `javac` and `java`.
Staying with `module-path` would eliminate few files from changeset at least.

Reply:
Hmmm... This message might not come from this project...

Reply:
Probably this improvement:
- apache/maven#2020

For me this does not seem right to verify message logged by other component :disappointed:

Reply:
Indeed, this message come from another component and had to be modified after https://github.com/apache/maven/pull/2020. This check was there before the refactoring. I presumed that the original author had a good reason.